lib/bolt/bolt_option_parser.rb

```diff
@@ -847,9 +847,6 @@ def initialize(options)
              "Available filters are 'all', 'failure', and 'success'.") do |rerun|
         @options[:rerun] = rerun
       end
-      define('--noop', 'See what changes Bolt will make without actually executing the changes.') do |_|
```

Comment:
I think this moved by accident?

Reply:
Specifically because I was making sure it showed up in the help text and couldn't find it because I didn't think to look under 'inventory options'. :|